packages/neuron-ui/src/containers/MainContent/actionCreators/networks.ts

```diff
@@ -0,0 +1,92 @@
+import { setNetwork } from '../../../services/UILayer'
+import { Network } from '../../../contexts/Chain'
+import { Routes, Message } from '../../../utils/const'
+import { saveNetworks, loadNetworks } from '../../../utils/localStorage'
+import { MainActions } from '../reducer'
+import { defaultNetworks } from '../../../contexts/Settings'
+
+const Testnet = defaultNetworks[0].name
+
+export default {
+  setNetwork: (network: Network) => {
+    setNetwork(network)
+    return {
+      type: MainActions.SetNetwork,
+      payload: network,
+    }
+  },
+  saveNetworks: (idx: number, networks: Network[], editorNetwork: Network, navTo: (path: string) => void) => {
+    if (!editorNetwork.name) {
+      return {
+        type: MainActions.ErrorMessage,
+        payload: {
+          networks: Message.NameIsRequired,
+        },
+      }
+    }
+    if (editorNetwork.name.length > 28) {
```

Comment:
Please make `28` a constant.

Reply:
updated in https://github.com/nervosnetwork/neuron/pull/166/commits/a3a5908cfa0a67cee9ff7c6f5c79906c16e9e5a7